javascript/ql/experimental/adaptivethreatmodeling/lib/experimental/adaptivethreatmodeling/EndpointFeatures.qll

```diff
@@ -128,23 +128,44 @@ private string getACallBasedTokenFeatureComponent(
 
 /** This module provides functionality for getting the function body feature associated with a particular entity. */
 module FunctionBodies {
+  string getTokenizedAstNode(ASTNode node) {
+    // NB: Unary and binary operator expressions e.g. -a, a + b and compound
+    // assignments e.g. a += b can be identified by the expression type.
+    result = node.(Identifier).getName()
+    or
+    // Computed property accesses for which we can predetermine the property being accessed.
+    // NB: May alias with operators e.g. could have '+' as a property name.
+    result = node.(IndexExpr).getPropertyName()
```

Comment:
These two I'm not 100% sure I get the examples. What does e.g. `a+b` end up as?

Reply:
The case on this line for property accesses, e.g. `response["date"] = new Date()` would get tokenized to `response date Date` (since we don't currently tokenize keywords like `new`).

Thanks for pointing out that the comments on the `Identifier` case don't really make sense — these are related to the old `CodeToFeatures` AST and are no longer relevant.